README.md

```diff
@@ -43,42 +54,30 @@ Easily invoke commands from the IBM Cloud CLI from directly inside of the VS Cod
 - Use the `ibmcloud login` command to log in to IBM Cloud (using your IBM Cloud credentials)
 
 ### Using `ibmcloud dev` workflow:
-Video demonstrating `ibmcloud dev` workflow: [IBM Cloud CLI Extension for VS Code
-](https://youtu.be/JmrLaLRRQSk)
 
 Steps to get started:
 - Create a project using one of the two methods below:
     - `ibmcloud dev create` using the IBM Cloud CLI (outside of VS Code)
     - [IBM Cloud CLI Documentation](https://cloud.ibm.com/docs/apps?topic=apps-create-deploy-app-cli)
-- Open the *project’s folder* locally in the VS Code editor
+- Open the *project’s folder* in VS Code
+    - Press Ctrl-K+Ctrl-O or navigate to File -> Open Folder to select folder
 - Use the `ibmcloud dev build` command to build the app into a Docker image
 - Use the `ibmcloud dev debug` command to run the app in local Docker for development
 - Use the `ibmcloud dev run` command to run the app in local Docker in release mode
 - Use the `ibmcloud dev deploy` command to deploy the app (in the Docker container) to IBM Cloud
 - Use the `ibmcloud dev view` command to open your deployed project on IBM Cloud in a web browser
-- Use the `ibmcloud dev console` command to open the IBM Cloud console for your project in a web browser
```

Comment:
I don't recall the discussion around removing `dev console`. Can you remind me why this was done?

Reply:
Sorry. I think that was a mistake. The code is still there but I must have accidentally remove this line from the README. I have readded. Thanks for catching that.